### PR TITLE
Adding Configuration

### PR DESCRIPTION
**Pull request by JoeHarrison**
Set up your own configuration.cfg in tygron-connect folder as such:
username: demousername
password: demopassword

### CODE REVIEW

File: tygron-environment/src/main/java/nl/tudelft/contextproject/tygron/TygronSettingsLoader.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+  String username = "";
+  String password = "";
+
+  /**
+   * Groups can individually decide what username they will fall back on if the
+   * loading or reading of the cfg file fails.
+   */
+  public TygronSettingsLoader(String path) {
+    try {
```
Comment (PauluzzNL) [inline tygron-environment/src/main/java/nl/tudelft/contextproject/tygron/TygronSettingsLoader.java]: Please move the try catch to the TygronSettings so it tries to load data through the settings loader, and if it fails, it should fall back to the default tygron settings.

**Comment by PauluzzNL** [comment]
:+1:

**Comment by PauluzzNL** [comment]
:+1:

**Comment by devbridie** [comment]
SettingsLoaderTest tests don't actually fail when the files can't be found.

**Comment by devbridie** [comment]
:+1:

**Comment by PauluzzNL** [comment]
:+1:

**Pull request merged**